.circleci/config.yml

```diff
@@ -33,7 +33,8 @@ jobs:
       - run:
           name: Lint Python
           command: |
-            flake8 --max-line-length 89 stdpopsim setup.py tests qc docs/_ext
+            flake8 --version
+            flake8 --max-line-length 89 stdpopsim setup.py tests docs/_ext
```

Comment:
I don't get it - isn't the ``qc`` directory still there?

Reply:
Its contents got moved under stdpopsim/qc, along side the stdposim/catalog folder.

Reply:
Ahhh. Good call.